source/gameengine/Ketsji/KX_Scene.cpp

```diff
@@ -2090,6 +2135,7 @@ PyAttributeDef KX_Scene::Attributes[] = {
 	EXP_PYATTRIBUTE_RW_FUNCTION("pre_draw", KX_Scene, pyattr_get_drawing_callback, pyattr_set_drawing_callback),
 	EXP_PYATTRIBUTE_RW_FUNCTION("post_draw", KX_Scene, pyattr_get_drawing_callback, pyattr_set_drawing_callback),
 	EXP_PYATTRIBUTE_RW_FUNCTION("pre_draw_setup", KX_Scene, pyattr_get_drawing_callback, pyattr_set_drawing_callback),
+	EXP_PYATTRIBUTE_RW_FUNCTION("on_exit", KX_Scene, pyattr_get_exit_callback, pyattr_set_exit_callback),
```

Comment:
I know that the other callbacks are named using `_`, but this will change (see the issue related to API renaming), so I think you should use camelCase : `onExit`.

Reply:
To be fair, I am kind of mixed here, but let's go with camelCase.